data-prepper-plugins/otlp-source/src/main/java/org/opensearch/dataprepper/plugins/source/otlp/OTLPSource.java

```diff
@@ -0,0 +1,180 @@
+    }
+
+    if (server == null) {
+      @SuppressWarnings("unchecked")
+      Buffer<Record<? extends Metric>> metricBuffer = (Buffer<Record<? extends Metric>>) (Object) buffer;
+
+      final OTelLogsGrpcService oTelLogsGrpcService = new OTelLogsGrpcService(
+          (int) (otlpSourceConfig.getRequestTimeoutInMillis() * 0.8),
+          otlpSourceConfig.getLogsOutputFormat() == OTelOutputFormat.OPENSEARCH ? new OTelProtoOpensearchCodec.OTelProtoDecoder() : new OTelProtoStandardCodec.OTelProtoDecoder(),
+          buffer, pluginMetrics);
+
+      final OTelMetricsGrpcService oTelMetricsGrpcService = new OTelMetricsGrpcService(
+          (int) (otlpSourceConfig.getRequestTimeoutInMillis() * 0.8),
+          otlpSourceConfig.getMetricsOutputFormat() == OTelOutputFormat.OPENSEARCH ? new OTelProtoOpensearchCodec.OTelProtoDecoder() : new OTelProtoStandardCodec.OTelProtoDecoder(),
+          metricBuffer, pluginMetrics);
+
+      final OTelTraceGrpcService oTelTraceGrpcService = new OTelTraceGrpcService(
+          (int) (otlpSourceConfig.getRequestTimeoutInMillis() * 0.8),
+          otlpSourceConfig.getTracesOutputFormat() == OTelOutputFormat.OPENSEARCH ? new OTelProtoOpensearchCodec.OTelProtoDecoder() : new OTelProtoStandardCodec.OTelProtoDecoder(),
+          buffer, pluginMetrics);
+
+      ServerConfiguration serverConfiguration = ConvertConfiguration.convertConfiguration(otlpSourceConfig);
+      CreateServer createServer = new CreateServer(serverConfiguration, LOG, pluginMetrics, "otlp", pipelineName);
+      CertificateProvider certificateProvider = null;
+      if (otlpSourceConfig.isSsl() || otlpSourceConfig.useAcmCertForSSL()) {
+        certificateProvider = certificateProviderFactory.getCertificateProvider();
+      }
+
+      List<GRPCServiceConfig<?, ?>> serviceConfigs = new ArrayList<>();
+      serviceConfigs.add(new GRPCServiceConfig<>(oTelLogsGrpcService, otlpSourceConfig.getLogsPath(),
+          LogsServiceGrpc.getExportMethod()));
+      serviceConfigs.add(new GRPCServiceConfig<>(oTelMetricsGrpcService, otlpSourceConfig.getMetricsPath(),
+          MetricsServiceGrpc.getExportMethod()));
+      serviceConfigs.add(new GRPCServiceConfig<>(oTelTraceGrpcService, otlpSourceConfig.getTracesPath(),
+          TraceServiceGrpc.getExportMethod()));
+
+      server = createServer.createGRPCServer(authenticationProvider, serviceConfigs, certificateProvider);
+
+      pluginMetrics.gauge(SERVER_CONNECTIONS, server, Server::numConnections);
+    }
+
+    try {
+      server.start().get();
+    } catch (ExecutionException ex) {
+      if (ex.getCause() != null && ex.getCause() instanceof RuntimeException) {
+        throw (RuntimeException) ex.getCause();
+      } else {
+        throw new RuntimeException(ex);
+      }
+    } catch (InterruptedException ex) {
+      Thread.currentThread().interrupt();
+      throw new RuntimeException(ex);
+    }
+    LOG.info("Started otlp_source on port " + otlpSourceConfig.getPort() + "...");
```

Comment:
```suggestion
    LOG.info("Started otlp source on port {}.", otlpSourceConfig.getPort());
```

Use Slf4J's string interpolation.

The source name is `otlp` so we don't need that underscore.

Also, there is no need for the final ellipses.